src/fmt.ml

```diff
@@ -65,6 +70,10 @@ let rec list ?(pp_sep = cut) pp_v ppf = function
 | v :: vs ->
     pp_v ppf v; if vs <> [] then (pp_sep ppf (); list ~pp_sep pp_v ppf vs)
 
+let hashtbl ?(pp_sep = cut) ~pp_k ~pp_v ppf tbl =
+  let f ppf k v = pf ppf "@[<1>%a:@.%a@]%a" pp_k k pp_v v pp_sep () in
+  Hashtbl.iter (f ppf) tbl
```

Comment:
> It's easier to add than to remove enclosing.

Yes. However, I'm always a bit sitting between two chairs with these combinators but tend to lean on the "just make it useful for printf debugging" one (if you want to do really really pretty things you can redefine the pretty printer) and it that case it's more annoying to have to define your own enclosing to ensure nice output. Lists are argueably a bit different as they often can represent text fragments.